src/stylesheets/components/_navigation.scss

```diff
@@ -35,15 +35,14 @@ $navigation-height: 50px;
 
 .app-navigation__link {
   @include govuk-typography-weight-bold;
-  text-decoration: none;
+  @include govuk-link-style-no-underline;
 
   &:not(:focus):visited {
     color: $govuk-link-colour;
   }
 
   &:not(:focus):hover {
     color: $govuk-link-colour;
```

Comment:
I'm sure there's a reason, just wondering why we don't want the link colour to change (get darker) on hover (which is the default behaviour)?

Reply:
Good question. That's the existing design, I don't _think_ we were looking to change it. But looping in @christopherthomasdesign in case he has any thoughts on whether we should be making a colour change for the navigation hover state?

Reply:
I just left it to be similar to how it behaves now. I think you’d have to ask someone before my time to remember why it’s like that! Maybe it’s because they don’t have underlines by default, so just adding the underline is enough of a visual change by itself? Worth mentioning that links in the header don't change colour on hover either, just add an underline. Could be some tacit rule about navigation links that I'm not aware of 😬

Reply:
No worries, it was just something I noticed. Happy for the links to be left as they are 👍

Reply:
I don't think there's any particular reason for it that I can remember. I think they're just inconsistent. In the [quicky hacky version](https://preview-link-style-updates--govuk-design-system-preview.netlify.app/) I did I removed pretty much all of the custom styling for the navigation links, so they did change colour on hover, if it's useful to compare.

Also happy for them to be left as they are, but maybe something to revisit later?